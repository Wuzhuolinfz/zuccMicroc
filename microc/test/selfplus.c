void main(int n) {
  n++;
  print("%d",n+=4);
  
}