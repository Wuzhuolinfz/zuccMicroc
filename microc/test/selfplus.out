24 19 1 5 25 0 4 13 13 11 0 4 1 12 15 -1 22 21 1